### ISSUE TITLE
SBCL.exe not found

### ISSUE THREAD

**Issue by AP2008** [roswell/roswell#474]
## Environment
Windows 10

## How to reproduce the issue

```sh
ros setup

```

## Expected behavior

Roswell install successfully

## Actual result

Roswell downloads the archive and extracts it. Then, this error pops up:
```sh
File not found - sbcl.exe
```

**Comment by Alf0nso** [comment]
Just came here to say that it happened the same to me, but when I go look into .roswell folder it does exist an executable.. just not on the path that is specified on the terminal, after running

![image](https://user-images.githubusercontent.com/32681068/115442188-a3b6be80-a209-11eb-83ea-344e997ec83e.png)

**Comment by AP2008** [comment]
@Alf0nso Even I have the executable in that folder

**Comment by Alf0nso** [comment]
@AP2008 Yes I know you have executable. Roswell does install it but after that it does not seem able to find the place where it left the executable.

I attempted at moving sbcl.exe and the other files into "C:\Users\user.roswell\src\sbcl-2.1.3-x86-64-windows\" but it produced the same effect. Like I said, I am still confident this will probably be a minor bug/error I might try to look into the source code to see if I can spot the mistake.

**Comment by Gwenio** [comment]
The problem is occurring when it is trying to move the files to the location it wants them.

The relevant source file is: [https://github.com/roswell/roswell/blob/master/src/install-sbcl-bin_windows.c](https://github.com/roswell/roswell/blob/master/src/install-sbcl-bin_windows.c)

I can't identify a clear source of the problem, though the commands being passed to the System() function to be executed seem weird.

---

If you move the files to the correct directories manually then roswell will recognize them.

Here are the correct directories (using placeholders for your username, the cpu architecture, and the SBCL version):

- `C:/Users/<username>/.roswell/impls/<arch>/windows/sbcl-bin/<version>/bin/sbcl.exe`
- `C:/Users/<username>/.roswell/impls/<arch>/windows/sbcl-bin/<version>/lib/sbcl/sbcl.core`
- `C:/Users/<username>/.roswell/impls/<arch>/windows/sbcl-bin/<version>/lib/sbcl/contrib`

**Comment by snmsts** [comment]
I think I fixed this problem with https://github.com/roswell/roswell/commit/f1555cd79cabf921cda4d313fc8a6ed7b5c8253b this commit. sorry I didn't notice issue but hope new release fix the problem.

**Comment by Alf0nso** [comment]
@snmsts please don't apologize! I will test it once I have time :)
